build.gradle

```diff
@@ -171,9 +171,15 @@ def versions = [
 ]
 
 dependencies {
+  // exclude old version of json-smart as it prevents the application from starting
+  def withoutJsonSmart = {
+    exclude group: 'net.minidev', module: 'json-smart'
+  }
+
   compile group: 'net.javacrumbs.shedlock', name: 'shedlock-spring', version: '1.0.0'
   compile group: 'net.javacrumbs.shedlock', name: 'shedlock-provider-jdbc', version: '1.0.0'
   compile group: 'com.microsoft.azure', name: 'azure-storage', version: '8.0.0'
+  compile group: 'com.microsoft.azure', name: 'azure-servicebus', version: '2.0.0-PREVIEW-5', withoutJsonSmart
```

Comment:
I think preview version should not be used in prod as it won't be stable. Could we use `1.2.7` instead ?

Reply:
Provided that this is a WIP, so I am afraid you'll have to review it again, I am going to address your comments for now.
Preview-5 should not be far from final release and we are not in prod yet, so this seems to me like a sensible choice, especially considered that 1.x has got old dependencies which should be avoided. Even 2.x for the time being has got a dependency which is not considered safe by the owasp dependency checker, but all considered it seems to me a better choice.

Reply:
Ok, let's use 1.2.7 for now and keep an eye on the update